drivers/bluetooth/hci/spi.c

```diff
@@ -300,12 +269,10 @@ static int bt_spi_send(struct net_buf *buf)
 		return -EINVAL;
 	}
 
+	spi_conf.operation |= SPI_HOLD_ON_CS;
```

Comment:
`~SPI_HOLD_ON_CS`?

Reply:
Not here. Below to the `&=`

Reply:
I am bit lost: before it was setting CS low, then trying to enable the chip, if so it kept it low, transceive, then putting it high. 

That's what I tried to do with cs_hold bit: on enablement it does not set CS high again, it's kept low. On next transceive it will put it high again.

Reply:
I was not clear. My second comment refers to my first comment. I meant to add the first comment on line 288 and not on line 272.